src/core/def/mpi.hpp

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
could be `phare_mpi.hpp` to really try to avoid any filename conflicts

todo, add header guards

Reply:
yeah ok